typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumSuperseedUSDCWarpConfig.ts

```diff
@@ -20,14 +23,39 @@ const owners = {
   solanamainnet: 'JAPPhnuChtzCGmskmFdurvAxENWwcAqXCV5Jn5SSiuWE',
 };
 
+const REBALANCER = '0xa3948a15e1d0778a7d53268b651B2411AF198FE3';
+
+const CONTRACT_VERSION = '8.0.0';
```

Comment:
i think it would be a good idea to have this constant exported from the SDK instead becuase it will probably be useful in other places too